### DISCUSSION TITLE
Add bounced subject to bounced notification email

### DISCUSSION THREAD

**Discussion by Metabaron1** [simple-login/app#452] [Feature Requests]
Hi
Most of bounced emails are fully available for download, but some bodies are missing. What is the trigger that delete the body, the size?
Is there a ways to keep all full bounced emails on self hosted, as it's just rejected by mailbox provider but not always spam, and it should not be lost...

And it could be intersting to add subjet of bouced email in the bounce notification email: 
An email sent to your alias ... from ... **with subjet:...** was bounced by your mailbox ...
It would avoid downloading and check the file if there no insterest.

Thanks

**Comment by nguyenkims** [comment]
A bounced email is deleted after 7 days but I think this isn't the cause for the issue you described here. The bounce email is retrieved from the bounce report (as SimpleLogin doesn't store emails) which is sent back to us by the mailbox service. Some services don't return the original email body that might explain this issue. Unfortunately returning the original email body is considered as best-practice but not obligatory and some mailbox services don't follow that convention.

> Is there a ways to keep all full bounced emails on self hosted, as it's just rejected by mailbox provider but not always spam, and it should not be lost...

If the full body isn't returned by the mailbox service, it isn't helpful to store the bounces I guess.

>  An email sent to your alias ... from ... with subjet:... was bounced by your mailbox ...

That's a good idea! We'll try to add it in the next version then :).

**Reply by Metabaron1** [reply]
Oh I thought it was your choice to delete some bodies. OK I understand.
Btw I still have all bounced email (6 months old), they are not deleted after 7 days. I don't know why it's not deleted but it's fine for me to keep them as it's not real spam, only rejected by mailbox.

If the full body isn't returned by the mailbox service, it could still be useful to keep it as it contains the error code and the reason why it's rejected. 

ok thanks :)

**Reply by nguyenkims** [reply]
> Btw I still have all bounced email (6 months old), they are not deleted after 7 days

Are you using SimpleLogin SaaS or the self-hosting? The bounced emails are indeed only deleted in the SaaS version as this isn't necessary for a self-hosted instance :)

**Reply by Metabaron1** [reply]
yes self-hosted. It's ok then  :)
